pkg/pool-linear/contracts/LinearPool.sol

```diff
@@ -29,8 +29,10 @@ import "./LinearMath.sol";
 import "./LinearPoolUserDataHelpers.sol";
 
 /**
- * @dev LinearPool suitable for assets with an equal underlying token with an exact and non-manipulable exchange rate.
- * Requires an external feed of these exchange rates.
+ * @dev Linear Pools are designed to hold two assets: "main" and "wrapped" tokens that have an equal value underlying
+ * token (e.g., DAI and aDAI).
```

Comment:
```suggestion
 * @dev Linear Pools are designed to hold two assets: "main" and "wrapped" tokens that have an equal value underlying
 * token (e.g., DAI and aDAI). The Pool will register three tokens in the Vault however: the two assets and the BPT
 * itself, so that BPT can be exchanged (effectively joining and exiting) via swaps.
 *
 * Unlike most other Pools, this one does not attempt to create revenue by charging fees: value is derived by holding
 * the wrapped, yield-bearing asset.
```

Reply:
I debated including that... ok; will add (as commit, so lint doesn't get upset)